server/src/main/java/org/jetbrains/bsp/bazel/server/sync/BazelProjectMapper.java

```diff
@@ -38,8 +38,9 @@ public BazelProjectMapper(
   public Project createProject(
       Map<String, TargetInfo> targets, Set<String> rootTargets, ProjectView projectView) {
     languagePluginsService.prepareSync(targets.values());
+    var dependencyTree = new DependencyTree(targets.values().toSet(), rootTargets);
```

Comment:
maybe do
var targetSet = targets.values().toSet() and pass it to prepareSync and DependencyTree to avoid multiple conversions

Reply:
`prepareSync` expects `Seq` should I replace it with `Set` and all transitive usages?